### PR TITLE
feat(group): added buttons to manage admin & members

### PR DESCRIPTION
**Pull request by FredericEspiau**

### CODE REVIEW

File: src/exchange/group/accounts/group-accounts.controller.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 angular.module("Module.exchange.controllers")
     .controller("ExchangeGroupAccountsCtrl", class ExchangeGroupAccountsCtrl {
-        constructor ($scope, Exchange, messaging, navigation, translator) {
+        constructor ($rootScope, $scope, Exchange, messaging, navigation, translator) {
             this.services = {
+                $rootScope,
```
Comment (AxelPeter) [inline src/exchange/group/accounts/group-accounts.controller.js]: Why this.services.$rootScope -> this.$rootScope, no ?
Same for $scope

Comment (Apollidore) [reply src/exchange/group/accounts/group-accounts.controller.js]: It's kind of a good practice to differenciate services and actual controller properties, which is something done in Angular 2+, but as I'm the only one to do it I guess I'll have to let it go

https://www.youtube.com/watch?v=moSFlvxnbgk

File: src/exchange/group/manager/group-manager.html [hunk 2]
```diff
@@ -57,4 +57,10 @@
              data-pagination-server-side-table-loading="getLoading()">
         </div>
     </div>
+
+    <div class="col-md-3 mt-5 mt-lg-0">
+        <button class="btn btn-block btn-default"
```
Comment (antleblanc) [inline src/exchange/group/manager/group-manager.html]: +`type="button"`

File: src/exchange/group/member/group-member.html [hunk 3]
```diff
@@ -57,4 +57,10 @@
              data-pagination-server-side-table-loading="getLoading()">
         </div>
     </div>
+
+    <div class="col-md-3 mt-5 mt-lg-0">
+        <button class="btn btn-block btn-default"
```
Comment (antleblanc) [inline src/exchange/group/member/group-member.html]: +`type="button"`

File: src/exchange/group/manager/group-manager.html (hunk 2 above)
Comment (FredericEspiau) [reply src/exchange/group/manager/group-manager.html]: Fixed

File: src/exchange/group/member/group-member.html (hunk 3 above)
Comment (FredericEspiau) [reply src/exchange/group/member/group-member.html]: Fixed

File: src/exchange/group/accounts/group-accounts.controller.js [hunk 4]
```diff
@@ -74,51 +74,51 @@ angular.module("Module.exchange.controllers")
...
                     const addGroupMessages = {
-                        OK: this.services.translator.tr("exchange_GROUPS_settings_success_message", this.selectedGroup.mailingListDisplayName),
-                        PARTIAL: this.services.translator.tr("exchange_GROUPS_settings_partial_message", this.selectedGroup.mailingListDisplayName),
-                        ERROR: this.services.translator.tr("exchange_GROUPS_settings_error_message", this.selectedGroup.mailingListDisplayName)
+                        OK: this.translator.tr("exchange_GROUPS_settings_success_message", this.selectedGroup.mailingListDisplayName),
+                        PARTIAL: this.translator.tr("exchange_GROUPS_settings_partial_message", this.selectedGroup.mailingListDisplayName),
+                        ERROR: this.translator.tr("exchange_GROUPS_settings_error_message", this.selectedGroup.mailingListDisplayName)
                     };
 
                     if (data == null) {
```
Comment (jleveugle) [inline src/exchange/group/accounts/group-accounts.controller.js]: What do you think about : https://lodash.com/docs/3.10.1#isEmpty ? More cases will be tested as `{}` `[]` `""` etc :)

File: src/exchange/group/manager/group-manager.controller.js [hunk 5]
```diff
@@ -12,8 +13,11 @@ angular
             };
 
             this.$routerParams = Exchange.getParams();
+            this.selectedGroup = angular.copy(navigation.selectedGroup);
 
-            $scope.$on(Exchange.events.accountsChanged, () => $scope.$broadcast("paginationServerSide.reload", "managersTable"));
+            $rootScope.$on(Exchange.events.accountsChanged, () => {
```
Comment (jleveugle) [inline src/exchange/group/manager/group-manager.controller.js]: Can we avoid using `$rootScope` please ?

File: src/exchange/group/manager/group-manager.controller.js [hunk 6]
```diff
@@ -24,29 +28,31 @@ angular
...
+            this.loading = true;
+
+            this.services
+                .group
+                .retrievingManagersByGroup(this.$routerParams.organization, this.$routerParams.productId, this.selectedGroup.mailingListName, count, offset)
+                .then((accounts) => {
+                    this.managersList = accounts;
+                })
+                .catch((failure) => {
+                    this.services.messaging.writeError(this.services.translator.tr("exchange_tab_ACCOUNTS_error_message"), failure);
```
Comment (jleveugle) [inline src/exchange/group/manager/group-manager.controller.js]: Indentation is broken :(

**Comment by jleveugle** [comment]
Can you redirect your PR to `develop`branch ?

**Pull request closed without merging**